mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java

```diff
@@ -104,6 +106,8 @@ public void handle(JournalService journalService, DataImportEventPayload eventPa
   private Future<JournalRecord> populateRecordTitleIfNeeded(JournalRecord journalRecord,
                                                             DataImportEventPayload eventPayload) {
     var entityType = journalRecord.getEntityType();
+    Optional.ofNullable(journalRecord.getTitle())
+      .ifPresentOrElse(title -> {}, () -> journalRecord.setTitle(NO_MARC_TITLE_MESSAGE));
```

Comment:
I think we can simplify this, as option StringUtils and java trinary operator

Reply:
Done